Automattic/wp-calypso
# Update purchase list view

#### Changes proposed in this Pull Request

This PR updates the design of the purchases listing screen with a more presentation that offers more helpful information:

**Before**
![image](https://user-images.githubusercontent.com/6981253/99919573-394a3280-2cec-11eb-8314-142fadbb3d6d.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/99920147-97c4e000-2cef-11eb-8dae-5d0811d200a0.png)

![image](https://user-images.githubusercontent.com/6981253/99919627-9645e880-2cec-11eb-8540-2178152420b6.png)

![image](https://user-images.githubusercontent.com/6981253/99920161-a8755600-2cef-11eb-8eaa-18316ecd83c3.png)


#### Testing instructions



Visit the site and account level purchases screen and test with the following products:

- Jetpack
- Monthly plan
- G Suite
- Domain
- Domain mapping
- Donations, or subscriptions from a site like Long reads or http://burlingtonbuddhist.org (note that we don't get payment method data from 3rd party sites. We'll need to add this in a separate PR)

Update store admin on one, or more products, to test the following scenarios:

- Expiring in 30 days (Should see a yellow warning)
- Expiring in 7 days (Should see a red warning)

Test with a credit card that is about to expire (this can be done while sandboxed by adding a credit card with an expiration date before the subscription end date. 

Lastly, take a look in some different browsers to see if you can spot anything.
